source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -332,7 +332,24 @@ export class CardDesignerSurface {
 
         if (this.isPreviewMode) {
             cardToRender.onExecuteAction = (action: Adaptive.Action) => {
-                alert("Action executed\n" + JSON.stringify(action.toJSON(this._serializationContext), undefined, 4));
+                const actionType = action.getJsonTypeName();
+                let message = `Action executed: "${actionType}" title: "${action.title}"`;
+
+                if (actionType === Adaptive.ExecuteAction.JsonTypeName) {
+                    const actionExecute = <Adaptive.ExecuteAction>action;
+                    message += ` verb: "${actionExecute.verb}"`;
+                }
+
+                if (actionType === Adaptive.OpenUrlAction.JsonTypeName) {
```

Comment:
I prefer `else if`

Reply:
I think because of the last if the code cannot be if {} else if {} else if{}, but he can reorder the conditionals to be something like

if (OpenUrlAction)
{
  ...
} 
else if (SubmitAction || ExecuteAction)
{
  if (ExecuteAction) {...}
  ...
}